File: c8/test2/test2.cpp
// test2.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include <stdio.h>
#include <string.h>

void length(char a[100],int *x);


int main(int argc, char* argv[])
{
	printf("ÊäÈëÈı¸ö×Ö·û´®£º\n");
	char a[100],b[100],c[100];
	gets(a);
	gets(b);
	gets(c);
	
	int *i,*j,*k;
	length(a,&i);
	length(b,&j);
	length(c,&k);

	return 0;
}



void length(char a[100],int *x)
{
	for(x=0;a[x]!='\0';i\x++);
}

File: java/j2/6.练习与实验（作业）/Test2/Test2.java
public class Test2{
	public static void main(String[] args){
		int i = 0;
		int j = 0;
		int sum = 0;
		int mut = 0;
		
		for(i = 1; i <= 10; i++){
			mut = i;
			
			for (j = 1; j < i; j++){
				mut *= j;
			}
			
			sum += mut;	
		}
		
		System.out.println("1!+ 2! + ，，，，，， + 10! = " + sum);
	}
}